charts/aws-ebs-csi-driver/templates/tests/helm-tester.yaml

```diff
@@ -193,10 +193,19 @@ metadata:
   annotations:
     "helm.sh/hook": test
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+    "ignore-check.kube-linter.io/run-as-non-root": "kubetest2 image runs as root"
+    "ignore-check.kube-linter.io/no-read-only-root-fs": "test pod requires privileged access"
+    "ignore-check.kube-linter.io/unset-cpu-requirements": "CPU limits on Kubernetes are an antipattern"
```

Comment:
/lgtm

I'll leave it up to you, but is it really worth catering to a linter that enforces a widely known antipattern?

Reply:
CPU resource limit check is being removed: https://github.com/stackrox/kube-linter/pull/712, putting up a new revision shortly that removes this annotation + memory limit given that one is enforced by test-infra pod.